code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -285,3 +285,22 @@
 		update_icon()
 
 		return FALSE
+
+/obj/item/weapon/material/brassknuckles/gold
+    default_material="gold"
+    name = "Brass Knuckles"
+    desc = "Brass Knuckles used on your fist for hard hits used in gangs"
+    icon_state = "brass"
+    item_state = "brass"
+    throw_speed = 7
+    throw_range = 9
+    force_divisor = 0.8 // 45 when wielded with hardnes 60 (steel)
+    thrown_force_divisor = 0.4 // 10 when thrown with weight 20 (steel)
+    slot_flags = SLOT_BELT | SLOT_BACK
+    block_chance = 0
+    attack_verb = list("thwacked", "hit", "bruised")
+    cooldownw = 8
+    value = 60
+
+/obj/item/weapon/material/sword/brassknuckles/gold
```

Comment:
I assume this was a mistake since you ported over the values from swords since this scope is in the wrong file, remove this relevant line please; can you clarify that you have tested your work also?